src/tracing/types.rs

```diff
@@ -96,7 +97,7 @@ pub struct CallTrace {
     /// Opcode-level execution steps.
     pub steps: Vec<CallTraceStep>,
     /// Optional complementary decoded call data.
-    pub decoded: DecodedCallTrace,
+    pub decoded: Option<Box<DecodedCallTrace>>,
```

Comment:
should we also make this an option @DaniPopes ?

I guess so because then we dont need to box two options